manim/scene/scene_file_writer.py

```diff
@@ -79,6 +81,14 @@ def __init__(self, renderer, scene_name, **kwargs):
         self.next_section(
             name="autocreated", type=DefaultSectionType.NORMAL, skip_animations=False
         )
+        # fail fast if ffmpeg is not found
+        if not ensure_executable(Path(config.ffmpeg_path)):
+            logger.critical(
+                """FFmpeg not found.
+Manim requires an ffmpeg installation to render videos.
+Please download ffmpeg """
```

Comment:
```suggestion
                "Manim could not find ffmpeg, which is required for generating "
                "video output. Please consult https://docs.manim.community/en/stable/installation.html "
                "for installation instructions, or set ffmpeg_path in Manim's configuration "
                "to the path of your ffmpeg executable."
```

I'm not entirely happy with my suggestion, but I'd like to include information on how to resolve the issue of ffmpeg not begin found.

Reply:
Actually the way I have set this up is like
```toml
[ffmpeg]
path = "path/to/ffmpeg"
```

How about:
```py
logger.critical(
"Manim could not find ffmpeg, which is required for generating video output."
"For installing ffmpeg please consult https://docs.manim.community/en/stable/installation.html"
"Make sure to either add ffmpeg to the PATH environment variable "
"or set path to the ffmpeg executable under the ffmpeg header in Manim's configuration."
```

Reply:
I feel that we should ask users interactively about where the location of the executable is. What do you think?

Reply:
+1 for interactivity in a follow-up.

With the current setup, is the ffmpeg executable_path being queried when `manim init` is run? That would be useful (and if not, please open an issue for it; follow-up too).

The proposed message looks okay, just a space after the URL is missing as far as I can tell. And it might be good to move the message from the logger to the `RuntimeError`, as proposed in the other comment.

Reply:
Done